### ISSUE TITLE
Accessibility: Incorrect tab navigation markup on product detail page

### ISSUE THREAD

**Issue by philippsauer** [intershop/intershop-pwa#1823]
**Actual Behavior**

Currently, confusing markup is used for the tab navigation on the PDP. The enclosing ul is marked as a tablist. The li within, however, is a regular ListItem (only the a in the li is marked as a tab role).

This results in two errors:
- The li no longer has an associated list. Explanation: the ul element loses its native semantic role as a list container because ARIA roles override native semantics. This means the li children are now orphaned from their expected parent role.
- The tablist has no valid children. Explanation: elements with role="tab" should be owned by the tablist container. When you have li elements in between, you create a semantic mismatch that can confuse screen readers.

**Expected Behavior**

The markup for the tab navigation on the PDP is valid. 

Recommendation 1
`<div role=”tablist”><button role=”tab” aria-controls=”tab1” aria-selected=”true/false”>….`

Recommendation 2 (optional):
Assign tabindex=”0” only to the active tab and the deactivated tabs (tabindex=”-1”). The user then tabs into the tablist and can change tabs using the left/right keys.

**Steps to Reproduce the Bug**

1. Go to PDP
2. Observe incorrect markup of tab navigation:
![Image](https://github.com/user-attachments/assets/f4b6e765-60ac-45ab-9901-b5c795509d09)

**Environment Details**
- Desktop:
  - OS: Win 11
  - Browser: Chrome latest
  - Project Code Version: 7.0.0

[AB#107367](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/107367)
